apps/frontend/src/components/navbar/burger-button/burger-button.tsx

```diff
@@ -2,11 +2,12 @@ import { component$ } from '@builder.io/qwik';
 
 export interface BurgerProps {
   buttonTitle: string;
+  onClick: () => void;
```

Comment:
Maybe we should pass the signal instead of a function?
Than we can just change the value when clicking inside the component.